website/docs/r/function_app.html.markdown

```diff
@@ -43,8 +43,8 @@ resource "azurerm_function_app" "example" {
   location                   = azurerm_resource_group.example.location
   resource_group_name        = azurerm_resource_group.example.name
   app_service_plan_id        = azurerm_app_service_plan.example.id
-  storage_account_name       = azurerm_storage_account.test.id
```

Comment:
Does the property with the name 'name' really needs a id value and not the name. Sounds a little bit inconsistent

Reply:
There is an other PR for this: https://github.com/terraform-providers/terraform-provider-azurerm/pull/6641

Reply:
You are right! I only replaced the label.